manifests/t/Toit/Toit/2.0.0-alpha.180/Toit.Toit.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.4.6 $debug=NVS1.CRLF.7-4-7.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: Toit.Toit
+PackageVersion: 2.0.0-alpha.180
+PackageLocale: en-US
+Publisher: Toit contributors
```

```suggestion
Publisher: Toitware ApS
```
This must match Control Panel.